docs/rewind/manual/licensing.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 9
+---
+
+# Licensing
+
+## Overview
+
+Rewind is available in three different editions, which differ only in the number of channels they
+can process:
+
+| Edition | Channels | RTA |
+|---------|----------|-----|
+| Personal | 16 | No |
+| Professional | 64 | Yes |
+| Production | 128 (wow!) | Yes |
+
+To protect your investment, guard against intellectual property theft, and safeguard our
+R&D for the future, we use the *iLok* dongle system to license Rewind. You will need an iLok 2 or
+iLok 3 to run *Rewind*.
+
+We chose *iLok* because we know many engineers will have one already, but if you don't, they can be
+obtained cheaply from your favourite rainforest-related eCommerce provider, or alternatively
+*Fourier Audio* can provide pre-licensed dongles ready for use.
+
+The first time you start *Fourier Rewind*, you'll be prompted to insert your *iLok* and
+enter your activation code to obtain your licence. After that, you should never need to think about
+it ever again; the licence simply lives on your *iLok*.
+
+All three editions run via the same *Rewind* application; if you find you need more channels, simply
+upgrade your licence - mid-gig if necessary! Upgrade pricing [is
+available](mailto:hello@fourieraudio.com).
+
+## iLok Login Details
```

Comment:
From this line on, terms like `iLok` and `Rewind` are no longer italicised. I don't have a preference between italics and non-italics, but it would be better to be consistent

Reply:
Done :)